cockatrice/src/player.cpp

```diff
@@ -698,13 +698,13 @@ void Player::retranslateUi()
     counterColors.append(tr("Green"));
 
     for (int i = 0; i < aAddCounter.size(); ++i){
-        aAddCounter[i]->setText(tr("&Add counter (%1)").arg(counterColors[i]));
+        aAddCounter[i]->setText(tr("%1").arg(counterColors[i]));
     }
     for (int i = 0; i < aRemoveCounter.size(); ++i){
-        aRemoveCounter[i]->setText(tr("&Remove counter (%1)").arg(counterColors[i]));
+        aRemoveCounter[i]->setText(tr("%1").arg(counterColors[i]));
     }
     for (int i = 0; i < aSetCounter.size(); ++i){
-        aSetCounter[i]->setText(tr("&Set counters (%1)...").arg(counterColors[i]));
+        aSetCounter[i]->setText(tr("%1...").arg(counterColors[i]));
```

Comment:
I'm wondering why exactly we have this `...` dots sometimes in the menus?
What's the intend/design behind them?

Because it's not completely uniform in all menus or submenus!